public/index.html

```diff
@@ -488,6 +491,9 @@ <h4 class="subtitle">
               <button class="button is-success" @click="getHtml('tandc_content','tandc_txtarea')">HTML</button>
               <button class="button is-success" @click="getMarkdown('tandc_content', 'tandc_txtarea')">Markdown</button>
               <button class="button is-success" v-if="contentRenderType!=1" @click="preview()">Preview</button>
+              <div style="flex:1">
+                <button class="button is-info" @click="fc_deploy()" style="float:right;">Deploy</button>
+              </div>
```

Comment:
Changes to this file are actually now generated via the PUG setup. I am hoping this change was a result of running the script that is inside this repo. Otherwise it will be be overwritten when I re-run it.

Reply:
You are correct - it was changed as part of the compilation. I guess it should have been omitted from the commit. Maybe the compiled files should be added to `.gitignore` if you're not using git for deployment?

Reply:
Yes. Thats valid. I will actually do that. Good shout out.